.circleci/config.yml

```diff
@@ -283,7 +283,14 @@ jobs:
     steps:
       - simulate_nested:
           task: "/sep/ink-002-holocene-system-config-upgrade"
-          
+
+  just_simulate_031_uni_holocene_systemconfig_update:
+    docker:
+      - image: <<pipeline.parameters.default_docker_image>>
+    steps:
+      - simulate_nested:
+          task: "/eth/031-uni-holocene-systemconfig-update"
+
   simulate_sequence:
```

Comment:
We also need to add the task 031 in the sim sequence **AFTER** the PR ( https://github.com/ethereum-optimism/superchain-ops/pull/683 ) is merged. 
We will have to merge and then add the 031 after the 030 to simulate in sequence.

Reply:
This one has been merged so we can add into the ci. 
I will make the suggestion